gradle/libs.versions.toml

```diff
@@ -22,7 +22,7 @@ mockito="4.11.0"
 nessieClientVersion = "0.71.0" # Must be in sync with Nessie version in the Iceberg release.
 nessieBuildPlugins = "0.2.24"
 opentelemetry = "1.31.0"
-opentelemetryAlpha = "1.25.0-alpha"
+opentelemetryAlpha = "1.31.0-alpha"
```

Comment:
Grmph the most recent version available is 1.30.1-alpha.

Reply:
Um - https://repo1.maven.org/maven2/io/opentelemetry/instrumentation/opentelemetry-instrumentation-bom-alpha/ shows 1.31.0-alpha :(

Reply:
Yea - that `-semconv` artifact is missing in 0.31.0-alpha :(